README.md

```diff
@@ -7,24 +7,23 @@
 [pprof][pprof-url] support for Node.js.
 
 ## Prerequisites
-1. Your application will need to be using Node.js 8.9.4 or greater, or Node.js 
-10.4.1 or greater. The profiler will not be enabled when using earlier versions
-of Node 8 and 10 because the profiler is not stable with those versions of
-Node.js.
+1. Your application will need to be using Node.js 8.9.4 or greater, Node.js 
+10.4.1 or greater, or any version of Node 12. The profiler should not be
+enabled when using earlier versions of Node 8 and 10 because the profiler is
```

Comment:
should not be - user should not do it
will not be - even if users tries to, it wont. 

I think "will not be" is the right phrase to use here.

Reply:
This change is intentional. 

This module does not currently check the Node.js version and prevent one from using the module based on the version. This strikes me as reasonable behavior -- for profiling for local testing, these stability issues likely aren't huge issues (if they were, they likely would have been identified in the V8 profilers earlier); preventing the usage of these profilers seems too restrictive.